examples/resources/snowflake_resource_monitor/resource.tf

```diff
@@ -1,14 +1,22 @@
-resource "snowflake_resource_monitor" "monitor" {
-  name         = "monitor"
+resource "snowflake_resource_monitor" "minimal" {
+  name                      = "resource-monitor-name"
+  credit_quota              = 100
+  notify_triggers           = [100]
+  suspend_immediate_trigger = 150
+  notify_users              = ["USERONE", "USERTWO"]
```

Comment:
These fields are not required by the schema, so they shouldn't be present in the minimal example. If this regards
> - Cannot create resource monitors with only triggers set, any other attribute has to be set.

I think we should add a comment here about it.

Reply:
Yeah, but I wanted to create minimal example that actually performs the job of the resource monitor. Without at least those fields the resource underneath wouldn't perform any action, but maybe that's something we should talk about how do we want to present our minimal examples. cc: @sfc-gh-asawicki

Reply:
Adjusted examples a bit to include miniminal minimal config with note

Reply:
That's a good point. I like these notes 👍